README.md

```diff
@@ -9,13 +9,14 @@ In order to use this workflow, you must already have completed the following ste
 2. Created an evaluation queue within the Synapse project.
 3. One or more Docker containers must have already been submitted to your evaluation queue.
```

Comment:
Can we add a link to the example template submission so that people know submissions have to be formatted a specific way?

Reply:
added in the latest commit